app/Shell/Docker.php

```diff
@@ -120,7 +120,9 @@ public function bootContainer(string $dockerRunTemplate, array $parameters): voi
         $process = $this->shell->exec('docker run -d --name "${:container_name}" ' . $dockerRunTemplate, $parameters);
 
         if (! $process->isSuccessful()) {
-            throw new Exception("Failed installing {$containerName}");
+            $containerName = array_key_exists('container_name', $parameters) ? $parameters['container_name'] : '';
+            $imageName = $parameters['image_name'];
+            throw new Exception("Failed installing {$imageName}. The container {$containerName} is already in use.");
```

Comment:
Is this the only reason we can have this failure? Feels like all we can do right now is just throw the error but with your corrected way of pulling the container name

Reply:
yes, the only reason is when a docker run fail. We can't catch the right container name.

The Docker class does not extend BaseService so we can't get containerName. 
I found this little "bug" when i wrote #123

Reply:
Right, but it's _possible_ the Docker run could fail for other reasons, right? Let me suggest an alternative.

Reply:
Yes, it is possible.
I changed the exception message to include the image name and a generic sentence about the container name because I don't know/i'm not sure how docker can fail.